src/main/java/com/hazelcast/web/HazelcastInstanceLoader.java

```diff
@@ -98,7 +98,14 @@ private static HazelcastInstance createClient(ClusteredSessionService sessionSer
             }
         }
         if (filterConfig.isStickySession()) {
-            clientConfig.getNetworkConfig().setConnectionAttemptLimit(1);
+            int initialBackoffMillis = clientConfig.getConnectionStrategyConfig()
+                    .getConnectionRetryConfig().getInitialBackoffMillis();
+            double multiplier = clientConfig
+                    .getConnectionStrategyConfig().getConnectionRetryConfig().getMultiplier();
+
+            // Limit connection attempts by 1
+            clientConfig.getConnectionStrategyConfig().getConnectionRetryConfig()
+                    .setMaxBackoffMillis(initialBackoffMillis * (int) multiplier);
```

Comment:
How is this going to make connection attempt limit to 1 which is previous value? Can you please add more explanation?

Reply:
For the values:

initialBackoff = 1000; multiplier=1.2; maxBackoff = ( 1000 * int(1.2) );

try1--wait 1000--try2--wait 1200--try3--wait 1440 ... up until waiting time exceeds the maxBackoff.

Since `maxBackoff = 1000 = 1st waiting time amount`, client stops attempting after the first trial.

However, now I noticed the attempt limit for the cases where `multiplier >= 2` becomes 2 (including the very first attempt done without waiting).

Reply:
ok